scripts/data_contamination/README.md

```diff
@@ -65,3 +65,15 @@ with beam.Pipeline() as pipeline:
 ## Notes
 
 The beam script does not support outputting contaminated ngrams yet.
+
+
+## Docker
+
+To create and run docker image:
+
+    docker build  . -t contamination_script
+    docker run -e INPUT_PATH=<input_path> -e SCENARIO_DATA=<scenario_data> -e OUTPUT_STATS=<output_stats> -e INPUT_FORMAT=<input_format> --rm -it  contamination_script:latest 
```

Comment:
I believe the caller also needs to use [bind mounts](https://docs.docker.com/storage/bind-mounts/) to allow the container to read from and write to the host filesystem.

Reply:
Yeah they'll need a bind mount or volume for their data. I added a quick note there as a FYI though I think the caller can determine the mechanism they want to use